src/sentry/eventstore/filter.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+
+class Filter:
```

Comment:
Should you extend object so that you don't get an [old-style class](https://wiki.python.org/moin/NewClassVsClassicClass)?

Reply:
Oops, thank you